nsxt/resource_nsxt_policy_transport_zone.go

```diff
@@ -19,14 +19,16 @@ import (
 
 var defaultInfraSitePath = "/infra/sites/default"
 
+var transportZonePathSample = "/infra/sites/[site]/enforcement-points/[enforcement-point]/transport-zones/[transport-zone]"
+
 func resourceNsxtPolicyTransportZone() *schema.Resource {
 	return &schema.Resource{
 		Create: resourceNsxtPolicyTransportZoneCreate,
 		Read:   resourceNsxtPolicyTransportZoneRead,
 		Update: resourceNsxtPolicyTransportZoneUpdate,
 		Delete: resourceNsxtPolicyTransportZoneDelete,
 		Importer: &schema.ResourceImporter{
-			State: resourceNsxtPolicyTransportZoneImporter,
+			State: getPolicyPathOrIDResourceImporter(transportZonePathSample),
```

Comment:
This importer won't work here, need that one which gathers `site_id`

Reply:
correct, my bad! thanks Kobi